backend/src/routes/helpers/validation.ts

```diff
@@ -29,4 +29,9 @@ function validateRequest<Output, TD extends z.ZodTypeDef, Input>(
   };
 }
 
-export { validateRequest };
+const stringToNumberTransformer = z
```

Comment:
Cleanup task after this is merged is to reuse this helper in the AllowlistEntry routes.